ngi_pipeline/utils/communication.py

```diff
@@ -13,10 +17,20 @@ def mail(recipient, subject, text, origin="ngi_pipeline"):
     s.sendmail('funk_002@nestor1.uppmax.uu.se', recipient, msg.as_string()) 
     s.quit()
 
+@with_ngi_config
 def mail_analysis(project_name, sample_name=None, engine_name=None,
                   level="ERROR", info_text=None, workflow=None,
                   recipient="ngi_pipeline_operators@scilifelab.se",
-                  subject=None, origin="ngi_pipeline"):
+                  subject=None, origin="ngi_pipeline",
+                  config=None, config_file_path=None ):
+
+    # check for mail recipient among the config values
+    # it is OK, if it is not configured, we are defaulting to ngi_pipeline_operators@scilifelab.se
+    try:
+        recipient = config.get('mail')['recipient']
+    except TypeError:
+        LOG.info("no mail recipient is defined in config file " + config_file_path + "; defaulting to " + recipient)
```

Comment:
if no recipient is found in the config we should skip sending mail. If I am not wrong right now the NGI-pipeline halts with this implementation...